packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -259,6 +277,24 @@ describe('c-quantic-search-box-input', () => {
             expect(suggestionsListItems.length).toEqual(
               mockSuggestions.length + exampleRecentQueries.length
             );
+
+            const suggestionOptionLabels =
+              suggestionsList.shadowRoot.querySelectorAll(
+                selectors.suggestionOptionText
+              );
+            const suggestionsAndRecentQueriesLength =
+              mockSuggestions.length + exampleRecentQueries.length;
+
+            expect(suggestionOptionLabels).not.toBeNull();
+            expect(suggestionOptionLabels.length).toEqual(
+              suggestionsAndRecentQueriesLength
+            );
+
+            suggestionOptionLabels.forEach((suggestion, index) => {
+              expect(suggestion.title).toEqual(
```

Comment:
is there a particular reason why you are testing only the last suggestion in the list?

Reply:
It was fixed here @SimonMilord but not above?

Reply:
oops forgot I changed it at 2 places, will address that